### PR TITLE
Changing http to https

### PR DESCRIPTION
**Pull request by jack17529**
Still in work - issue https://github.com/internetarchive/openlibrary/issues/324

### CODE REVIEW

**Comment by mekarpeles** [comment]
Is this ready to be merged @jack17529? so far LGTM

**Comment by jack17529** [comment]
There are still many places more to change.

**Comment by jack17529** [comment]
@mekarpeles  as there are more then 100 changes it will be simple for me if you keep merging my PRs

File: Readme.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ For instructions on administrating your Open Library instance and
 build instructions for developers, refer the Developer's
 [Quickstart.md](Quickstart.md) document.
 
-You can find more info digging into this old (and in part outdated) document here: http://code.openlibrary.org/en/latest/
+You can find more info digging into this old (and in part outdated) document here: https://code.openlibrary.org/en/latest/
```
Comment (hornc) [inline Readme.md]: the code.openlibrary.org domain is not actually secure. Where is it hosted, and is it something we should still be maintaining?

Comment (jack17529) [inline Readme.md]: Yeah https://code.openlibrary.org/en/latest/ is not secure.

**Comment by mekarpeles** [comment]
I'm partial to punting on any changes to code.openlibrary.org as it's likely on the horizon to being deprecated. I'd rather we move the valuable parts into our github README.md and into our Wiki.
Any complaints to this approach?

**Comment by bfalling** [comment]
I agree we should deprecate code.openlibrary.org.

**Comment by mekarpeles** [comment]
Can't merge as is -- https://code.openlibrary.org/en/latest/ insecure. Needs to be reverted to http in the meantime.

**Comment by bfalling** [comment]
@jack17529: I think we just have that one change mentioned by @mekarpeles needed in order to merge, right?

**Comment by mekarpeles** [comment]
Closing for now unless interest is resumed

**Pull request closed without merging**